pennylane_lightning/lightning_tensor/lightning_tensor.py

```diff
@@ -117,8 +117,6 @@
         "OrbitalRotation",
         "QFT",
         "ECR",
-        "BlockEncode",
-        "C(BlockEncode)",
```

Comment:
Why is this removed?

Reply:
We only have limited support for BlockEncode and Controlled ones, as tested before. With more tests were in, we found BlockEncode are not fully supported. Will work on this to get all the gate support later when working on statevector to MPS transform.